kotlin/src/main/java/app/rive/runtime/kotlin/RiveAnimationView.kt

```diff
@@ -142,11 +142,14 @@ open class RiveAnimationView(context: Context, attrs: AttributeSet? = null) :
                 val loopIndex = getInteger(R.styleable.RiveAnimationView_riveLoop, 3)
                 val autoplay =
                     getBoolean(R.styleable.RiveAnimationView_riveAutoPlay, defaultAutoplay)
+                val riveTraceAnimations =
+                    getBoolean(R.styleable.RiveAnimationView_riveTraceAnimations, false)
                 val artboardName = getString(R.styleable.RiveAnimationView_riveArtboard)
                 val animationName = getString(R.styleable.RiveAnimationView_riveAnimation)
                 val stateMachineName = getString(R.styleable.RiveAnimationView_riveStateMachine)
                 val resourceId = getResourceId(R.styleable.RiveAnimationView_riveResource, -1)
                 val url = getString(R.styleable.RiveAnimationView_riveUrl)
+                renderer = RiveArtboardRenderer(autoplay = autoplay, trace=riveTraceAnimations)
```

Comment:
Kotlin formatter also might want a pass here. I think it's `CMD + Alt + L` in Android Studio